mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -488,10 +488,19 @@ private class LogisticAggregator(
     this
   }
 
+  /**
+   * @return The numbers of training samples.
```

Comment:
Though this is a private class, the doc should be `Returns the numbers of training samples.` in order to show up automatically as the method summary in the generated doc. Users have to make one click to see `return ...`